docs/admin-guide/admin-configuration.md

```diff
@@ -64,7 +64,9 @@ With a full (not minimal) zot image, the following extension features can be ena
     "sync": {},
     "search": {},
     "scrub": {},
-    "lint": {}
+    "lint": {},
+    "trust": {},
```

Comment:
Wasn't this called "imagetrust"? we really need an example under zot../examples for this, maybe in the "all extensions" one

Reply:
No, when we discussed about moving this to a separate extension you explicitly mentioned it should be called only 'trust' in the config.